### PR TITLE
add examples as integration tests

### PR DESCRIPTION
**Pull request by andre-merzky**

### CODE REVIEW

**Comment by andre-merzky** [comment]
This is ready for review - the failing analytics tests will be addressed by #3221.

**Comment by andre-merzky** [comment]
> @mtitov proposed to move the examples into a separate scheduled CI.

This is implemented now.  @mtitov - does the setup look reasonable to you?

File: .github/workflows/ci-release.yml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+
+name: CI
```
Comment (mtitov) [inline .github/workflows/ci-release.yml:2]: maybe to keep a different name for this action? i.e., to differentiate it from the main CI with tests (`Run examples`?)

Comment (andre-merzky) [reply .github/workflows/ci-release.yml]: Thanks, done.

**Review by AymenFJA — APPROVED** [review]
I left one comment; besides that, it looks good to me.

**Pull request merged**